setup.py

```diff
@@ -42,11 +42,13 @@
       test_suite='nose.collector',
       packages=['qiita_core',
                 'qiita_db',
+                'qiita_db/handlers',
                 'qiita_db/metadata_template/',
-                'qiita_pet',
+                'qiita_pet/',
```

Comment:
typo because I was testing exactly that. Turns out it doesn't matter if it has slash or no slash.